core/libraries/form_sections/inputs/EE_Email_Confirm_Input.input.php

```diff
@@ -1,10 +1,11 @@
 <?php
 /**
- * EE_Email_Input
+ * EE_Email_Confirm_Input
  *
  * @package         Event Espresso
  * @subpackage
- * @author              Mike Nelson
+ * @since           4.10.4
```

Comment:
You can put `$VID:$` in for new code. When the release is built the version number will get automatically swapped in.